ParabolInc/parabol
# feat(jira-server): List issues in sprint poker

Resolves #6096

Based on #6092

![image](https://user-images.githubusercontent.com/466991/156552579-be53525f-60cb-4519-b713-6a493e9bfdac.png)


How to test:

* Setup Jira Server integration
* Create poker meeting
* Open Jira Server tab in scope phase
* See a list of issues loaded
